app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/RepresenteeController.scala

```diff
@@ -167,9 +170,13 @@ class RepresenteeController @Inject() (
   }
 
   def enterId(): Action[AnyContent] = authenticatedActionWithSessionData.async { implicit request =>
-    withCapacitorOrPersonalRepresentativeAnswers(request) { (_, journey, answers) =>
+    withCapacitorOrPersonalRepresentativeAnswers(request) { (representativeType, journey, answers) =>
       val backLink = answers.fold(
-        _ => routes.RepresenteeController.enterName(),
+        _ =>
+          representativeType.fold(
+            _ => routes.RepresenteeController.enterDateOfDeath(),
+            _ => routes.RepresenteeController.enterDateOfDeath()
```

Comment:
it should be the enterName page for capacitors

Reply:
🤦